src/wrappers/themis/jsthemis/secure_session.cpp

```diff
@@ -36,8 +37,11 @@ int SecureSession::get_public_key_for_id_callback(
         return THEMIS_BUFFER_TOO_SMALL;
     }
     v8::Local<v8::Value> argv[1] = {Nan::CopyBuffer((char*)id, id_length).ToLocalChecked()};
-    v8::Local<v8::Value> a =
-        Nan::Call(((SecureSession*)(user_data))->id_to_pub_key_callback_, 1, argv).ToLocalChecked();
+    Nan::MaybeLocal<v8::Value> result = Nan::Call(reinterpret_cast<SecureSession*>(user_data)->id_to_pub_key_callback_, 1, argv);
```

Comment:
looks like magic to me ✨

Reply:
Previously we have asserted that no error can possibly happen here and unwrapped the `MaybeLocal` reference unconditionally (it's still there, just as a temporary value). Well, in fact, an error *is possible* here if the callback throws. Now we check for that before unwrapping the value to avoid triggering an assertion failure.